cypress/integration/task_actions.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types="Cypress" />
+
+describe("Task Action Buttons", function() {
+  before(() => {
+    cy.login();
+  });
+  beforeEach(() => {
+    cy.preserveCookies();
+  });
+
+  it("Some buttons should be disabled and others should be clickable. Clicking on buttons produces banners.", () => {
+    cy.visit(
+      "/task/evergreen_ubuntu1604_test_model_patch_5e823e1f28baeaa22ae00823d83e03082cd148ab_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48/logs"
+    );
+    cy.dataCy("schedule-task").should("have.css", "pointer-events", "none");
+    cy.dataCy("restart-task").click();
+    cy.dataCy("banner").contains("Task scheduled to restart");
+    cy.dataCy("ellipsis-btn").click();
+    cy.dataCy("unschedule-task").click();
+    cy.dataCy("banner").contains("Task marked as unscheduled");
+    cy.dataCy("ellipsis-btn").click();
+    cy.dataCy("abort-task").should("have.css", "pointer-events", "none");
+    cy.dataCy("prioritize-task").click();
+    cy.get(".ant-input-number-input")
+      .clear()
+      .type("99");
+    cy.get(".ant-btn.ant-btn-primary.ant-btn-sm")
```

Comment:
can we use a `data-cy` selector on this. Targeting these elements based on class names is error prone if theres another element with the same class on the page.

Reply:
I wasn't able to find a way to apply data-cy to an ant component